source/DataAccess/DADGenerator.ts

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+import IntegrationConfigFactory from '../IntegrationConfigFactory';
+import { IIntegrationConfig, IntegrationType } from '../IIntegrationConfig';
+import { mkdir, writeFile } from 'fs';
+
+export default class DataAccessDoc {
+
+    private _integrationType;
+    private _queries;
+    private _header: string[] = [];
+    private _footer: string[] = [];
+
+    constructor(integrationType: string) {
+        this._integrationType = integrationType;
+        this._header.push(`# Data Access Requirements: ${this._integrationType}`);
+        this._header.push('The following tables and fields are required for the DIA with integrationConfig: ' + this._integrationType);
+        this._footer.push('');
+
+        const icf = new IntegrationConfigFactory();
+        const cfg: IIntegrationConfig = icf.create(this._integrationType);
+
+        this._queries = cfg.queries;
+    }
+
+    public create(): void {
+        const path = './docs/DataAccess/' + this._integrationType + '.md';
+        const data = this._header.join('\n') + '\n\n' + this._createTable().join('\n') + '\n\n' + this._footer.join('\n');
```

Comment:
Assuming \n is part of the github approved markdown spec you could place the value in a const somewhere and refer to it in this line.